doc/devel/color_changes.rst

```diff
@@ -0,0 +1,133 @@
+.. _color_changes:
+
+*********************
+Default Color changes
+*********************
+
+As discussed at length elsewhere [insert links], ``jet`` is an
+empirically bad color map and should not be the default color map.
+Due to the position that changing the appearance of plot is a back
+compatibility breaking change, this change has been put off for far
+longer than it should have been.  In addition to changing the default
+color map we plan to take the chance to change the default color-cycle
+on plots and to adopt a different color map for filled plots (``imshow``,
+``pcolor``, ``contourf``, etc) and for scatter like plots.
+
+
+Default Heat Map Colormap
+-------------------------
+
+The choice of a new color map is fertile ground to bike-shedding ("No,
+it should be _this_ color") so we have a proposed set criteria (via
+Nathaniel Smith) to evaluate proposed color maps.
+
+- it should be a sequential colormap, because diverging colormaps are
+  really misleading unless you know where the "center" of the data is,
+  and for a default colormap we generally won't.
```

Comment:
There is https://github.com/matplotlib/matplotlib/pull/3858 which adds a normalizer for dealing with that situation.

Picking anything about the diverging color map (if to use it, where to center it, if the upper/lower limits should be symmetric) requires knowing something about the _meaning_ of the data which is a business mpl should not be in (as it is too widely used).  This sort of domain-specific defaults should be the job of down-stream packages (ex scikit-image, pandas, seaborn, ...)